xbmc/video/VideoInfoTag.h

```diff
@@ -189,6 +189,17 @@ class CVideoInfoTag : public IArchivable, public ISerializable, public ISortable
   virtual bool IncrementPlayCount();
 
   /*!
+  * @brief Reset playcount to VALUE_NOT_SET
+  */
+  virtual void ResetPlayCount();
+
+  /*!
+  * @brief Check is the playcount is set to VALUE_NOT_SET
```

Comment:
No need to refer `VALUE_NOT_SET`, just say what it does: `Check if play count is set.`

`@return true if play count is set otherwise false`